apps/aehttpserver/config/prod.exs

```diff
@@ -1,23 +1,22 @@
 use Mix.Config
 
-# For production, we configure the host to read the PORT
-# from the system environment. Therefore, you will need
-# to set PORT=80 before running your server.
+# For production, we often load configuration from external
+# sources, such as your system environment. For this reason,
+# you won't find the :http configuration below, but set inside
+# Aehttpserver.Web.Endpoint.load_from_system_env/1 dynamically.
+# Any dynamic configuration should be moved to such function.
 #
-# You should also configure the url host to something
-# meaningful, we use this information when generating URLs.
+# Don't forget to configure the url host to something meaningful,
+# Phoenix uses this information when generating URLs.
 #
 # Finally, we also include the path to a cache manifest
 # containing the digested version of static files. This
 # manifest is generated by the mix phoenix.digest task
 # which you typically run after static files are built.
-config :aehttpserver, Aehttpserver.Endpoint,
-  server: true,
-  http: [port: 4000],
-  debug_errors: true,
-  code_reloader: true,
-  check_origin: false,
-  watchers: []
+config :aehttpserver, Aehttpserver.Web.Endpoint,
+  on_init: {Aehttpserver.Web.Endpoint, :load_from_system_env, []},
+  url: [host: "example.com", port: 80],
+  cache_static_manifest: "priv/static/cache_manifest.json"
```

Comment:
why do we reference a cache_manifest for static files, if we only provide an api?